src/main/java/Employee.java

```diff
@@ -0,0 +1,96 @@
+import javax.persistence.*;
+
+@Entity
+@Table(name = "employee")
+public class Employee {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+
+    @Column(name = "first_name")
+    private String first_name;
+    @Column(name = "last_name")
+    private String last_name;
+    @Column(name = "gender")
+    private String gender;
+    @Column(name = "age")
+    private int age;
+     @Column(name = "city_id")
+    private int city_id;
+
+    public Employee(){
+
+    }
+
+    public Employee(int id, String first_name, String last_name, String gender, int age,int city_id ) {
```

Comment:
```suggestion
    public Employee(String first_name, String last_name, String gender, int age,int city_id ) {
```
Убери из конструктора id. Хибернейт сам его проставит.